Specification Registry/Attributes/date.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I feel we would need separate fields for each of the use cases mentioned in the array; to begin with we can probably only have `dateOfBirth` and remove `age`

Reply:
Got it. Makes sense. Will make the necessary changes